administration/hot-reload.md

```diff
@@ -39,6 +39,12 @@ For using curl to reload fluent-bit, users must specify an empty request body as
 $ curl -X POST -d {} localhost:2020/api/v2/reload
 ```
 
+For Windows, to represent empty body, users have to use `'{}'` as a body instead of raw `{}`:
```

Comment:
```suggestion
On Windows, use `'{}'` to represent an empty body instead of raw `{}`:
```

Reply:
To be fair you could do this on Linux too I think?

Reply:
No, it's Windows specific.

Reply:
No, I mean you can quote it on Linux as well. It works fine so maybe we should just use the one format for both?

Both work for Linux:
```shell
$ curl -X POST -d {} 127.0.0.1:2020/api/v2/reload
{"reload":"done","status":0}
$ curl -X POST -d '{}' 127.0.0.1:2020/api/v2/reload
{"reload":"done","status":0}
```

Reply:
Ah, OK. We can do that. Let's move to use quoted empty body.